README.md

```diff
@@ -139,6 +139,7 @@ The default sorting for stories found is `state.position:asc,id:asc`, which tran
     -y, --type [name]        Set type of story
     --from-git               Fetch story parsed by ID in current git branch
     --git-branch             Create and checkout git branch from story (<type>-<id>-<slug>)
+    --git-branch-standard    Create git branch from story slug <mention-name>/ch<id>/<title> as required by the Git integration: https://bit.ly/2RKO1FF
```

Comment:
What do you think of changing this argument from `--git-branch-standard` to `--git-branch-ch`? It may better convey the target of this branch naming convention (and it's shorter).

Reply:
Yeah, I use the old format.

I am compelled, though, by the fact that this _is_ a client for Clubhouse, so we should _probably_ adhere to their favored branch naming convention.

Really, the only part I miss from their convention is the story `type`. What about a compromise? We could drop the new `--git-branch-standard` argument and instead change `--git-branch` to produce the output `<menion-name>/ch<id>/<type>-<slug>`.

What do you think?

Reply:
I am cool with that. I updated the PR.